ReactViewControl/ReactViewRender.LoaderModule.cs

```diff
@@ -74,6 +74,14 @@ internal class LoaderModule {
                 ExecuteLoaderFunction("loadDefaultStyleSheet", JavascriptSerializer.Serialize(NormalizeUrl(ViewRender.ToFullUrl(stylesheet.ToString()))));
             }
 
+            /// <summary>
+            /// Set the specified stylesheet.
+            /// </summary>
+            /// <param name="stylesheet"></param>
+            public void SetDefaultStyleSheet(ResourceUrl stylesheet) {
+                ExecuteLoaderFunction("setDefaultStyleSheet", JavascriptSerializer.Serialize(NormalizeUrl(ViewRender.ToFullUrl(stylesheet.ToString()))));
```

Comment:
create an aux function to serialize the stylesheet url and use it in the loadDefaultStyleSheet too